docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,86 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
```

Comment:
This isn't rendered correctly, at least from my browser:

![screen shot 2016-11-28 at 10 57 04 am](https://cloud.githubusercontent.com/assets/920884/20681680/794e7134-b559-11e6-8e85-d16e3330cdd4.png)